sci-mathematics/jags/jags-4.0.0.ebuild

```diff
@@ -0,0 +1,47 @@
+# Copyright 1999-2013 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=5
+
+inherit autotools-utils toolchain-funcs
+
+MYP="JAGS-${PV}"
+
+DESCRIPTION="Just Another Gibbs Sampler for Bayesian MCMC simulation"
+HOMEPAGE="http://mcmc-jags.sourceforge.net/"
+SRC_URI="mirror://sourceforge/project/mcmc-jags/JAGS/4.x/Source/${MYP}.tar.gz"
+LICENSE="GPL-2"
+IUSE="doc"
+
```

Comment:
Justin Lecher vas escriure el dia dv, 30 oct 2015:

> > +# Copyright 1999-2013 Gentoo Foundation
> > +# Distributed under the terms of the GNU General Public License v2
> > +# $Id$
> > +
> > +EAPI=5
> > +
> > +inherit autotools-utils toolchain-funcs
> > +
> > +MYP="JAGS-${PV}"
> > +
> > +DESCRIPTION="Just Another Gibbs Sampler for Bayesian MCMC simulation"
> > +HOMEPAGE="http://mcmc-jags.sourceforge.net/"
> > +SRC_URI="mirror://sourceforge/project/mcmc-jags/JAGS/4.x/Source/${MYP}.tar.gz"
> > +LICENSE="GPL-2"
> > +IUSE="doc"
> > +
> 
> Please move the empty line two lines up.

Excuse my ignorance, but which of the empty lines do you refer to? Between
$Id$ and EAPI?, between EAPI and inherit? I have been trying to look to
other ebuilds in the sci-mathematics and they seem to follow the same
lines. In fact, I have just copied this from jags-3.4.0.ebuild.

-- 
-  Xavier  -